etc/docker-scripts/bootstrapper.py

```diff
@@ -65,24 +72,26 @@ def __init__(self, **kwargs: Any) -> None:
         self.cos_endpoint = urlparse(self.input_params.get('cos-endpoint'))
         self.cos_bucket = self.input_params.get('cos-bucket')
         # TODO(check hardcoded false)
-        self.cos_client = minio.Minio(self.cos_endpoint.netloc,
-                                      access_key=os.getenv('AWS_ACCESS_KEY_ID'),
-                                      secret_key=os.getenv('AWS_SECRET_ACCESS_KEY'),
-                                      secure=False)
+        self.cos_client =\
+            minio.Minio(self.cos_endpoint.netloc,
+                        access_key=os.getenv('AWS_ACCESS_KEY_ID'),
+                        secret_key=os.getenv('AWS_SECRET_ACCESS_KEY'),
+                        secure=False)
 
     @abstractmethod
     def execute(self) -> None:
         """Execute the operation relative to derived class"""
-        raise NotImplementedError("Method 'execute()' must be implemented by subclasses!")
+        raise NotImplementedError("Method 'execute()' must be implemented'"
```

Comment:
Could you please update your linter to not be destructive?  The project already configures a linter.  We can talk about adjusting its configuration in another issue/pr if you want.

All of these reformats make it difficult to glean the actual changes of the pull request.